lib/utils.js

```diff
@@ -202,7 +202,12 @@ function prettifyMessage ({ log, messageFormat, messageKey = MESSAGE_KEY, colori
       if (p1 && log[p1]) {
         return log[p1]
       }
-      return ''
+      return p1.split('.').reduce(function (prev, curr) {
+        if (prev && prev[curr]) {
```

Comment:
this will have some problems if we have `curr` as a `length` property or anything else that is present on a string. I think we should use a for loop instead of a reduce and just break/return early

Reply:
fixed it to loop and return early